symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -0,0 +1,61 @@
+package com.symphony.bdk.core;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.auth.AuthenticatorFactory;
+import com.symphony.bdk.core.auth.OboAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.service.Obo;
+import com.symphony.bdk.core.client.ApiClientFactory;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.service.V4MessageService;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apiguardian.api.API;
+
+/**
+ * BDK entry point.
+ */
+@Slf4j
+@API(status = API.Status.EXPERIMENTAL)
+public class SymphonyBdk {
+
+  private final BdkConfig config;
```

Comment:
some class attributes can be instanced in the methods, except if you think these class will be inherits by other classes which can use them